.coveragerc

```diff
@@ -3,6 +3,10 @@ branch = True
 include = snekbox/*
 omit = snekbox/api/app.py
 
+[run]
+branch = True 
+omit = snekbox/config_pb2.py
+
```

Comment:
You're duplicating the entire `run` section, which I don't think is correct. Instead, you should append the path to the existing `omit` on line 4. To specify multiple omit values, you can put them on separate lines, as shown in the example in the documentation: https://coverage.readthedocs.io/en/6.2/source.html#execution

Reply:
i am not understanding how to  add multiple omit values do you have any other examples.

Reply:
Look at how `exclude_lines` is specified currently. Notice how each value is on a new line. The same would apply to `omit` for what you're trying to add.

Reply:
you mean like these omit = 
                                              snekbox/api/app.py
                                              snekbox/config_pb2.py

Reply:
Yes, though please indent the lines with 4 spaces, just like the values for `exclude_lines` are indented.